### DISCUSSION TITLE
Migrating from Old library )est at lib) to this new lib (lwesp)

### DISCUSSION THREAD

**Discussion by Moo2017** [MaJerle/lwesp#82] [Q&A]
Hi @MaJerle 
I've ported the old library and now I noticed that you have changed the library. I was wondering how difficult would be to migrate from the old one to this new one and could you please let me know what is the reason so change that? Is this new one more reliable and robust than the old version?

**Comment by MaJerle** [comment]
`esp-at-lib` has been renamed to `lwesp` first, to distinguish from *esp-at* library by Espressif. This has been done in the period of v0.xx releases where API may change.

The core architecture is the same and works at the same principle.
